<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="5893" started-at="2022-03-30T21:38:56 IST" finished-at="2022-03-30T21:39:02 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="5893" started-at="2022-03-30T21:38:56 IST" finished-at="2022-03-30T21:39:02 IST">
      <class name="com.testngpractice.Test_NG_Practice">
        <test-method status="PASS" signature="browser_Launch()[pri:0, instance:com.testngpractice.Test_NG_Practice@7f13d6e]" name="browser_Launch" duration-ms="5831" started-at="2022-03-30T21:38:56 IST" finished-at="2022-03-30T21:39:02 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- browser_Launch -->
        <test-method status="FAIL" signature="website_Launch()[pri:0, instance:com.testngpractice.Test_NG_Practice@7f13d6e]" name="website_Launch" duration-ms="4" started-at="2022-03-30T21:39:02 IST" finished-at="2022-03-30T21:39:02 IST">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.testngpractice.Test_NG_Practice.website_Launch(Test_NG_Practice.java:31)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- website_Launch -->
        <test-method status="PASS" signature="amazon()[pri:2, instance:com.testngpractice.Test_NG_Practice@7f13d6e]" name="amazon" duration-ms="3" started-at="2022-03-30T21:39:02 IST" finished-at="2022-03-30T21:39:02 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- amazon -->
      </class> <!-- com.testngpractice.Test_NG_Practice -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
